docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -216,14 +239,14 @@ Location | Description |
 
    - Option 2: Run `kubectl apply -f` on each of the ingress YAML files that are already included in the sample source `/tmp/mii-sample/ingresses` directory:
 
-   ```
-   $ cd /tmp/mii-sample/ingresses
-   $ kubectl apply -f traefik-ingress-sample-domain1-admin-server.yaml
-   $ kubectl apply -f traefik-ingress-sample-domain1-cluster-cluster-1.yaml
-   $ kubectl apply -f traefik-ingress-sample-domain1-cluster-cluster-2.yaml
-   $ kubectl apply -f traefik-ingress-sample-domain2-cluster-cluster-1.yaml
-   $ kubectl apply -f traefik-ingress-sample-domain2-cluster-cluster-2.yaml
-   ```
+       ```
+       $ cd /tmp/mii-sample/ingresses
+       $ kubectl apply -f traefik-ingress-sample-domain1-admin-server.yaml
+       $ kubectl apply -f traefik-ingress-sample-domain1-cluster-cluster-1.yaml
+       $ kubectl apply -f traefik-ingress-sample-domain1-cluster-cluster-2.yaml
+       $ kubectl apply -f traefik-ingress-sample-domain2-cluster-cluster-1.yaml
+       $ kubectl apply -f traefik-ingress-sample-domain2-cluster-cluster-2.yaml
+       ```
 
    > **NOTE**: We give each cluster ingress a different host name that is decorated using both its operator domain UID and its cluster name. This makes each cluster uniquely addressable even when cluster names are the same across different clusters.  When using `curl` to access the WebLogic domain through the ingress, you will need to supply a host name header that matches the host names in the ingress.
 
```

Comment:
Found another typo: For more on information ingresses -> For more information on ingresses

Reply:
Fixed. Thanks!